client/editEvents.js

```diff
@@ -160,13 +160,13 @@ Template.editEvents.helpers({
     }
     return this.size > maxAllowedRoundSize;
   },
-  roundComplete: function() {
+  progressClasses: function() {
     var progress = RoundProgresses.findOne({roundId: this._id});
-    return progress.isComplete();
-  },
-  roundOvercomplete: function() {
-    var progress = RoundProgresses.findOne({roundId: this._id});
-    return progress.isOverComplete();
+    var classes = "progress-bar-" + {incomplete: "warning", complete: "success", overcomplete: "danger"}[progress.completeness()];
```

Comment:
Looks like good cleanup. It's kind of unfortunate to be mixing the view logic up in this code, but since handlebars is so neutered, I imagine this is a lot cleaner than what we had.